content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -72,4 +73,6 @@ volumes to be attached to a node. For other instance types on
 <a href="https://aws.amazon.com/ec2/">Amazon Elastic Compute Cloud (EC2)</a>,
 Kubernetes would permit 39 volumes to be attached.
 
+* On Azure, up to 64 disks could be attached to a node, depending on the node type. You could refer to [Sizes for virtual machines in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes).
```

Comment:
"can be attached"

"For more details, refer to"

Reply:
fixed